commitments/op_commitment.go

```diff
@@ -0,0 +1,90 @@
+package commitments
+
+import (
+	"fmt"
+	"log"
+)
+
+type OPCommitmentType byte
```

Comment:
knit - suffixing `commitment` inside of all package files is redundant with package name. Could suffice as `commitments/op.go`, `commitments/eigenda.go`, etc

Reply:
np